packages/lesswrong/components/comments/CommentsNode.tsx

```diff
@@ -190,14 +192,14 @@ const CommentsNode = ({
   }
 
   const scrollIntoView = useCallback((behavior: "auto"|"smooth"="smooth") => {
-    if (!isInViewport()) {
+    if (!isInViewport() || hasInContextLinks) {
```

Comment:
Unrelated to the stuttering issue, to avoid the comment jumping just after you click it due to the permalink loading at the top